sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/RecognizeHealthcareEntityOptions.java

```diff
@@ -10,7 +10,7 @@
 import static com.azure.ai.textanalytics.implementation.Utility.DEFAULT_POLL_INTERVAL;
 
 /**
- * The {@link RecognizePiiEntityOptions} model.
+ * The {@link RecognizeHealthcareEntityOptions} model.
```

Comment:
Following the rename for `/RecognizePiiEntityOptions.java → .RecognizePiiEntitiesOptions.java`
Should this also be `RecognizeHealthcareEntitiesOptions`?

Reply:
I leave all healthcare changes in the healthcare PR. This change will cause build to fail so I made it.